docs/contribute.rst

```diff
@@ -396,4 +557,4 @@ tag name.
     docker run --rm --privileged tonistiigi/binfmt:latest --install arm64
     docker buildx create --name imgbldr --use
     docker buildx inspect --bootstrap --builder imgbldr
-    docker buildx build --build-arg REVISION='git sha' --build-arg ROTKI_VERSION=vx.x.x --file ./Dockerfile --platform linux/amd64 --platform linux/arm64 --tag rotki/rotki:vx.x.x --tag rotki/rotki:latest --push .
\ No newline at end of file
+    docker buildx build --build-arg REVISION='git sha' --build-arg ROTKI_VERSION=vx.x.x --file ./Dockerfile --platform linux/amd64 --platform linux/arm64 --tag rotki/rotki:vx.x.x --tag rotki/rotki:latest --push .
```

Comment:
Why is this line shown in the diff?

Reply:
was probably missing or not missing newline and emacs fixed it.